napari_allencell_annotator/util/file_utils.py

```diff
@@ -42,7 +53,7 @@ def is_supported(file_path: Path) -> bool:
         return extension in SUPPORTED_FILE_TYPES
 
     @staticmethod
-    def get_sorted_files_in_dir(dir_path: Path) -> list[Path]:
+    def get_sorted_dirs_and_files_in_dir(dir_path: Path) -> list[Path]:
```

Comment:
it seems a bit weird to me that we allow this function to give us invalid files and we filter for it later- lets refactor this:
- create function get_valid_images_sorted()
- this will return the valid images sorted (so we filter out invalid files immediately, and do this with one function call)

Reply:
refactored in https://github.com/aics-int/napari-allencell-annotator/pull/79/commits/dc378f83cc4bdd2e93e38a617b18b50de8a6eb0e

Reply:
invalid files should be filtered immedately, and not passed into `_add_selected_files()`. `_add_selected_files()` adds all images assuming the images are already valid.